packages/rrweb/src/replay/index.ts

```diff
@@ -571,10 +618,7 @@ export class Replayer {
     this.wrapper = document.createElement('div');
     this.wrapper.classList.add('replayer-wrapper');
     this.config.root.appendChild(this.wrapper);
-
-    this.mouse = document.createElement('div');
-    this.mouse.classList.add('replayer-mouse');
-    this.wrapper.appendChild(this.mouse);
+    // this.createPointer(-1);
```

Comment:
is this needed? when i added it, it created a mouse/pointer showing in the top left corner that did nothing

Reply:
it seems to work OK without it (tested on web, data without `pointerId`s, and data with `pointerId`). i think since we have the checks later in the code that create a pointer with id `-1` if the `pointerId` doesn't exist, we don't need this default one here

Reply:
It's not *needed*, but it maintains the existing behavior and mirrors your desktop where the mouse is always present